ui-v2/tests/acceptance/dc/kvs/delete.feature

```diff
@@ -0,0 +1,16 @@
+@setupApplicationTest
+Feature: dc / kvs / delete: KV Delete
+  Scenario: Delete ACL
```

Comment:
Completely agree, I'm still toying with how they should be named (both `Features` and `Scenarios`) so that you get a good DX. Notice the ember like way I've started calling the Features after the path of the feature file so it's clear where you need to be when it fails.

All in all these are 'just' comments really - but you're right, ideally they'd be more verbose and 'use case' led 👍